src/components/domain/user/userProfile/UserProfile.tsx

```diff
@@ -0,0 +1,38 @@
+import BorderCard from '@/components/common/card/BorderCard';
+import CirclePlanetIcon from '../CirclePlanetIcon';
+
+export default function UserProfile() {
+  // NOTE: api 연동 이후 수정 예정
+  const labels = ['이메일', '관심 직무', '기술 스택'];
+  const items = [
+    'PRism@gmail.com',
+    '디자이너',
+    'Figma, Protipie, Adobe XD, Adobe Photoshop, Figma, Figma, Protipie, Adobe XD, Adobe Photoshop, Figma',
+  ];
+
+  return (
+    <div className="flex w-full">
```

Comment:
```suggestion
    <div className="flex w-full gap-4">
```

+하단 div에 mr-4 제거

이 부분은 다음에 수정해주셔도 됩니다!